processor/error/package_tests/TestProcessErrorAugmentedUserAgentAndIP.approved.json

```diff
@@ -12,7 +12,7 @@
                 },
                 "user": {
                     "ip": "127.0.0.1",
-                    "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5)",
+                    "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5)",
```

Comment:
Seems like this would be a breaking change. I don't think anyone is using it at the moment, so we can probably move forward.

Reply:
It's not a breaking change as it hasn't been released yet.

Reply:
I'd rather see it as a bugfix, as we convert all other values to use '_'.